src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_common.py

```diff
@@ -83,12 +85,15 @@ def firewall_rule_create_func(client, resource_group_name, server_name, firewall
 def migration_create_func(cmd, client, resource_group_name, server_name, properties, migration_name=None):
 
     subscription_id = get_subscription_id(cmd.cli_ctx)
-
+    properties_filepath = os.path.join(os.path.abspath(os.getcwd()), properties)
+    if not os.path.exists(properties_filepath):
+        raise FileOperationError("Properties file does not exist in the given location")
+    with open(properties_filepath, "r") as f:
+        json_data = f.read()
```

Comment:
Better to utilize existing function `get_file_json` and `shell_safe_json_parse` in `azure.cli.core.util`.
See the same example in storage module:
https://github.com/Azure/azure-cli/blob/228fd1c8e749a3fbbf32593e47bdc8ca91ddb9b2/src/azure-cli/azure/cli/command_modules/storage/operations/account.py#L587-L591

Reply:
I'll see if the above 2 functions can be used . We want to intentionally remove the use of @ symbol since that is a point of potential failure for our customers. Passing in simple strings is much easier.